.travis.yml

```diff
@@ -1,10 +1,14 @@
 language: node_js
 node_js:
-  - "0.12"
-  - "1"
-  - "2"
-  - "3"
-  - "4"
+  - stable
+  - 4
+  - 3
+  - 2
+  - 1
+  - 0.12
+matrix:
+  allow_failures:
+    - node_js: stable
```

Comment:
If Node 6 comes out and breaks things it might be annoying to get failure emails, but also good to have the error logs in Travis, hence this PR.